site/content/blog/2020-05-04-pritunl.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "From OpenVPN to Pritunl VPN: The transition"
+slug: pritunl
+date: 2020-05-04T12:00:00-04:00
+author: Angelos Kyratzakos
+github: angeloskyratzakos
+community: angelos.kyratzakos
+---
+
+Usually companies are using office's network to prevent unauthorized people to connect to their private network and by using their own network infrastructure/connectivity they can maintain their desirable level of security for their data. But it would be convenient to connect to "that private network" while they are away from office's building while using an internet connection. To solve that problem, VPN (Virtual Private Network)
+is used to allow authorized remote access to company's private network.
+
+Working in a [fully remote company as Mattermost is](https://docs.mattermost.com/process/working-at-mattermost.html#work-anywhere), creates the need for employees to use a VPN connection in order to be able to access internal private infrastructure and resources. A vast majority of companies use OpenVPN as a solution to host those VPN connections on their own servers. [OpenVPN](https://en.wikipedia.org/wiki/OpenVPN) is a widely used software and protocol which was also our selection to be used as a quick and reliable solution to access our internal infrastructure. After using it a while we needed a better solution in terms of:
+- High Availability (HA)
+- Auditing
+- Supporting better access control (e.g. only SREs can access production servers and only developers can access development accounts)
+- Working with SSO, particularly OneLogin
+After some investigation, we ended up with [Pritunl](https://pritunl.com/) as:
+- it is built on the OpenVPN protocol,
+- it is [open source](https://github.com/pritunl/pritunl),
+- Pritunl client can be installed on any platform,
+- its pricing is reasonable for our scale,
+- hides the complete configuration (user and server management) overhead behind the scenes of the web interface.
```

Comment:
and this one as well, thank you very much for your help @justinegeffen

Reply:
No problem at all, happy to help. :)